### PR TITLE
fix: PoolTogether Invalid Countdown

### PR DESCRIPTION
**Pull request by iRhonin**
Changes
  - Fix countdown shows the wrong date when input seconds is zero
  - Fix If no wallet connected, not showing "connect a wallet" button once you open the invest pop up

closes #

### CODE REVIEW

**Comment by septs** [comment]
@DimensionDev/qa plz test

**Review by Jack-Works — DISMISSED** [review]

**Review by guanbinrui — APPROVED** [review]

**Comment by iRhonin** [comment]
I noticed another bug, let me fix that and reopen after.

**Comment by wenluomask** [comment]
For the latest branch pull/3875, polygon's pools can not be showed if I change to Matic Mainnet.

**Review by Jack-Works — DISMISSED** [review]

**Comment by iRhonin** [comment]
> For the latest branch pull/3875, polygon's pools can not be showed if I change to Matic Mainnet.

Fixed it.

**Review by Jack-Works — DISMISSED** [review]

File: packages/maskbook/src/plugins/PoolTogether/constants.ts [hunk 1]
```diff
@@ -14,3 +15,5 @@ export const ONE_DAY_SECONDS = 86400
 export const ONE_WEEK_SECONDS = ONE_DAY_SECONDS * 7
 
 export const ADDRESS_ZERO = '0x0000000000000000000000000000000000000000'
+export const SECONDS_PER_BLOCK = 14
+export const WEI_PER_ETHER = new BigNumber(10).pow(18)
```
Comment (UncleBill) [inline packages/maskbook/src/plugins/PoolTogether/constants.ts:19]: I don't see these two constant being used. Do you forget to remove them?

Comment (iRhonin) [reply packages/maskbook/src/plugins/PoolTogether/constants.ts]: Yes, I forgot, removed them.

File: packages/maskbook/src/plugins/PoolTogether/UI/DepositDialog.tsx [hunk 2]
```diff
@@ -305,7 +307,15 @@ export function DepositDialog() {
                             </Grid>
                             <Grid item>
                                 <Typography variant="body2" fontWeight="fontWeightBold" className={classes.oddsValue}>
-                                    {t('plugin_pooltogether_odds_value', { value: odds.toLocaleString() })}
+                                    {t('plugin_pooltogether_odds_value', {
+                                        value: odds.toLocaleString(),
+                                        period:
+                                            prizePeriodSeconds === ONE_DAY_SECONDS
```
Comment (septs) [inline packages/maskbook/src/plugins/PoolTogether/UI/DepositDialog.tsx:313]: as function?

File: packages/maskbook/src/plugins/PoolTogether/utils.ts [hunk 3]
```diff
@@ -55,3 +56,11 @@ export const getNetworkColor = (chainId: ChainId) => {
...
     }
 }
+
+export const getPrizePeriod = (t: I18NFunction, preiod: number) => {
+    return preiod === ONE_DAY_SECONDS
+        ? t('daily')
+        : preiod === ONE_WEEK_SECONDS
+        ? t('weekly')
+        : t('days', { preiod: (preiod / ONE_WEEK_SECONDS).toFixed() })
+}
```
Comment (septs) [inline packages/maskbook/src/plugins/PoolTogether/utils.ts:66]:
```suggestion
export function getPrizePeriod(t: I18NFunction, preiod: number) {
    if (preiod === ONE_DAY_SECONDS) {
        return t('daily')
    } else if (preiod === ONE_WEEK_SECONDS) {
        return t('weekly')
    }
    return t('days', { preiod: (preiod / ONE_WEEK_SECONDS).toFixed() })
}
```

**Review by Jack-Works — APPROVED** [review]

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**